tests/rptest/services/redpanda.py

```diff
@@ -549,11 +549,16 @@ def __init__(self,
             self._cloud_storage_bucket = f'panda-bucket-{uuid.uuid1()}'
             self.cloud_storage_url_style = cloud_storage_url_style
 
-            self.cloud_storage_api_endpoint = cloud_storage_api_endpoint
+            # Endpoint defaults to minio-s3
+            self.cloud_storage_api_endpoint = 'minio-s3'
```

Comment:
nit: I believe this method would be easier/more natural to read if it was something like this:

```
if cloud_storage_api_endpoint is not None:
  # set override ...
elif gcp:
  # gcp custom config
else:
  # default
```

or at least

```
# set default
if override:
  # override if requested
elif gcp:
  # or do gcp override
```

now it reads like this

```
# set default...
# maybe set gcp
# oh forget that, maybe override all we just did
```

Reply:
Edited this logic to be more natural per your first suggestion.